WordPress/src/main/java/org/wordpress/android/ui/uploads/PostUploadNotifier.java

```diff
@@ -146,6 +146,17 @@ void addPostInfoToForegroundNotification(@NonNull PostModel post, @Nullable List
         startOrUpdateForegroundNotification(post);
     }
 
+    void removePostInfoFromForegroundNotification(@NonNull PostModel post, @Nullable List<MediaModel> media) {
+        sNotificationData.totalPostItems--;
```

Comment:
Can we call `removePostInfoFromForegroundNotificationData` here?
It seems almost the the same code...


Also, not related to this PR, what do you think on remove the method
```
    void prepareForegroundNotificationForRetry(@NonNull PostModel post, @Nullable List<MediaModel> media) {
        removePostInfoFromForegroundNotificationData(post, media);
    }
```

and if you think is necessary rename `removePostInfoFromForegroundNotificationData`  ?

Not sure why we have a method that does nothing, except to call another method with the same signature.

Reply:
thank you for noticing 👀 
addressed in d554953